### DISCUSSION TITLE
Dynamic variable not being read

### DISCUSSION THREAD

**Discussion by destinyThompson** [emailjs-com/help#162] [Q&A] [answered]
I'm having an issue displaying dynamic content in my email template! The template shows the rows are created for each cycle of the loop however, all values are showing as empty strings. Has anyone had this issue before or knows where I'm going wrong?

My emailjs template has:
```
  {{#items}} <!-- Loop through items array -->
    <tr>
      <td class="table-item">
        <span style="font-family: georgia, palatino, serif;">{{items.name}}</span>
      </td>
      <td class="table-item">
        <span style="font-family: georgia, palatino, serif;">QTY: {{items.quantity}}</span>
      </td>
      <td class="table-item">
        <span style="font-family: georgia, palatino, serif;">${{items.unitPrice}}</span>
      </td>
    </tr>
  {{/items}} <!-- End loop -->
```

Example value received by emailjs for items: 
items: [ { name: "Wire A", unitPrice: "19.99", quantity: 2 }, { name: "Wire B", unitPrice: "9.99", quantity: 1 } ]

**Comment by xr0master** [comment, marked as answer]
You shouldn't use the `items` keyword inside the block.

**Reply by destinyThompson** [reply]
Not using the items keyword yields the same result unfortunately.

**Reply by destinyThompson** [reply]
I found the issue! It was within my JS. After that fix along with not using the items keyword within the block, it's working!!
